### PR TITLE
added categories with static data

### PR DESCRIPTION
**Pull request by JantuDeb**
#3 
### Category Card

Added an Error and Loading Component
Added Axios to load data from the API
image

### CODE REVIEW

**Review by ozairkhan2810 — APPROVED** [review]
looks good
liked how you stored image names in array of objects

**Review by Raghav888 — APPROVED** [review]
Good naming convention

**Review by mariyasada — APPROVED** [review]
liked how you broke down small components even Iconwrapper and error component for error

File: src/utils/axios-instance.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+// Create instance
+let instance = axios.create(defaultConfig);
+
+// Set the AUTH token for all request
+instance.interceptors.request.use(function (config) {
+  const token = localStorage.getItem("token");
+  config.headers.Authorization = token ? `Bearer ${token}` : "";
+  return config;
+});
+
```
Comment (tushartiwari7) [inline src/utils/axios-instance.js:19]: I liked this way of automating the `token` process on private routes. 👍

File: src/components/shared/IconWrapper.jsx [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+
+const IconWrapper = ({children}) => {
+  return (
+    <span className='flex center icon'>
+        {children}
+    </span>
+  )
+}
```
Comment (tushartiwari7) [inline src/components/shared/IconWrapper.jsx:9]: Liked how you made small components for things which you are going to reuse alot in future.

**Review by tushartiwari7 — APPROVED** [review]
Your code seems very optimal and liked how you made small components for `error` and `IconWrapper`. Just remove the comments from the code and youcan merge!

**Pull request merged**